app/controllers/courses_controller.rb

```diff
@@ -1,6 +1,12 @@
 class CoursesController < ApplicationController
   def create
+    if !current_user.admin
```

Comment:
This is a rails thing, but you can actually put a question mark after the admin:
```ruby
if !current_user.admin?
```